Tests/NIOSSLTests/TLSConfigurationTest.swift

```diff
@@ -1258,7 +1274,7 @@ class TLSConfigurationTest: XCTestCase {
         try assertHandshakeSucceeded(withClientConfig: clientConfig, andServerConfig: serverConfig)
     }
     
-    func testTLSPSKFailure() throws {
+    func testTLSPSKFailureDeprecated() throws {
```

Comment:
These tests should be annotated with `@available(*, deprecated)` to suppress warnings.